interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props are references in react that link specific data from the parent class to the child components.
 
-  Researched answer:
+  Researched answer: A keyword in React used for passing info between parent components to child components. 
```

Comment:
Correct props are part of the child component. specifically a part of the child component that is assigned by the parent component.